manager/controllers/app/fybrikapplication_controller.go

```diff
@@ -553,24 +562,27 @@ func (r *FybrikApplicationReconciler) constructDataInfo(req *datapath.DataInfo,
 	configEvaluatorInput.Request = CreateDataRequest(input, req.Context, &req.DataDetails.ResourceMetadata)
 
 	// Governance actions
-	err = r.checkGovernanceActions(configEvaluatorInput, req, appContext, env)
+	governanceMsg, err = r.checkGovernanceActions(configEvaluatorInput, req, appContext, env)
 	if err != nil {
-		return err
+		return "", errors.WithMessage(err, governanceMsg)
```

Comment:
we include governaceMsg in the error response, but lose the catalogMsg. Is it correct?

Reply:
The message from the policy manager might be relevant to understand the cause of the Deny action, for example.
Messages from catalog and policy manager are both included if there are no errors to construct the data plane. Otherwise, errors are not accumulated.

Reply:
Please add this explanation into the code, it can be relevant if somebody else would need to work with the code.

Reply:
Done